tools/depends/target/ffmpeg/FFMPEG-VERSION

```diff
@@ -1,5 +1,5 @@
 LIBNAME=ffmpeg
 BASE_URL=https://github.com/xbmc/FFmpeg/archive
-VERSION=3.1.6-Krypton
+VERSION=release/3.3-kodi
```

Comment:
ffmpeg 3.3 is required. versions are here:
https://github.com/FFmpeg/FFmpeg/commit/e1cc7f83df001b471313fb7f03fd4b0fa1c5a747